011-quarkus-panache-rest-flyway/src/test/java/io/quarkus/qe/containers/PostgreSqlDatabaseTestResource.java

```diff
@@ -0,0 +1,45 @@
+package io.quarkus.qe.containers;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class PostgreSqlDatabaseTestResource implements QuarkusTestResourceLifecycleManager {
+
+    public static final int PORT = 5432;
+
+    private static final String QUARKUS_DB_KIND = "quarkus.datasource.db-kind";
+    private static final String QUARKUS_DB_JDBC_URL = "quarkus.datasource.jdbc.url";
+    private static final String QUARKUS_DB_USER = "quarkus.datasource.username";
+    private static final String QUARKUS_DB_PASSWORD = "quarkus.datasource.password";
+    private static final String DEFAULT_SCHEMA = "test";
+    private static final String POSTGRESQL = "postgresql";
+
+    private PostgreSQLContainer<?> container;
+
+    @Override
+    public Map<String, String> start() {
+        container = new PostgreSQLContainer<>("postgres:10.5");
```

Comment:
We should use newer version, 13.1 or 12.5 ... this is my last comment to this :)

Reply:
Changes. Thanks!